library/sources/http-server/shouldDiscoverRoute.ts

```diff
@@ -61,11 +63,6 @@ function shouldDiscoverExtension(segment: string) {
 }
 
 function isDotFile(segment: string) {
-  // See https://www.rfc-editor.org/rfc/rfc8615
-  if (segment === ".well-known") {
```

Comment:
This check can stay no? We want to ignore the other .well-known files still right ("fake" .well-known files)

Reply:
I guess this check was always unnecessary haha